src/server/models/users.model.js

```diff
@@ -4,22 +4,32 @@ import mongoose from 'mongoose';
 
 const Schema = mongoose.Schema;
 let userSchema = new Schema({
-  userId: String,
-  email: String,
-  name: String,
-  age: Number,
-  field: String,
-  region: String,
-  skills: String,
-  regDate: {
-    type: Date,
-    default: Date.now(),
-  },
+    userId: String,
+    email:
+    { type: String,
+        unique: true,
+        required: true
+    },
+    platform_id: String,
+    platform_type:
+    { type: Number,
```

Comment:
actually what's the convention for putting { into next line? On line 24, { is on the same line as regDate. We should be consistent.